profile.md

```diff
@@ -55,6 +60,73 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
 
 - *Application*: The software that uses a Service Instance via Service Binding.
 
+## Originating Identity Header
+
+In the [Open Service Broker API specification](spec.md) it defines an
+additional HTTP Header that can be included in messages from the Platform
+to identify the user that requested the action to be taken.
+
+The header consists of two parts: a `platform` string and `value` string.
+Both values will vary based on the Platform which is being used. The
+following sections define the values to be used based on the Platform
+and which properties are expected to appear in the JSON encoded
+`value`.
+
+Note that when both the originating identity HTTP Header and the Context
+object appear in the same message the `platform` value MUST be the same
+for both.
+
+### Cloud Foundry
+
+*`platform` Value*: `cloudfoundry`
+
+The following properties MUST appear within the JSON encoded `value`:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| user_id | string | The `user_id` value from the Cloud Foundry JWT token. |
+
+Platforms MAY include additional properties.
+
+For example, a `value` of:
+```
+{
+  "user_id": "683ea748-3092-4ff4-b656-39cacc4d5360"
+}
+```
+would appear in the HTTP Header as:
+```
+X-Broker-API-Originating-Identity: cloudfoundry eyANCiAgInVzZXJfaWQiOiAiNjgzZWE3NDgtMzA5Mi00ZmY0LWI2NTYtMzljYWNjNGQ1MzYwIiwNCiAgInVzZXJfbmFtZSI6ICJqb2VAZXhhbXBsZS5jb20iDQp9
+```
+
+### Kubernetes
+
+*`platform` Value*: `kubernetes`
+
+The following properties MUST appear within the JSON encoded `value`:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| username | string | The `username` property from the Kubenernetes `user.info` object. |
+| uid | string | The `uid` property from the Kubenernetes `user.info` object. |
+| groups | string | The `groups` property from the Kubenernetes `user.info` object. |
```

Comment:
the `user.Info` object is an internal interface available to resthandlers in the request context, populated by the authentication layer. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/authentication/user/user.go#L19-L42

it can be serialized as https://kubernetes.io/docs/api-reference/v1.7/#userinfo-v1-authentication (or the v1beta1 struct you referred to), or you can define your own type that serializes the same four fields.